### ISSUE TITLE
DP_INFER_BATCH_SIZE

### ISSUE THREAD

**Issue by JQ-JQQ** [deepmodeling/deepmd-kit#5175]
### Summary

I used the parameter `DP_INFER_BATCH_SIZE=1024` to train on a dataset and then performed active learning using dpgen, resulting in a training set of 8000 structures and a dp potential (trained for 800,000 steps). The RMSE and dpgen accuracy within the target range of 100k-1000k met my requirements (99%).

Using the same training set (containing 8000 structures) and `DP_INFER_BATCH_SIZE=102400` (trained for 800,000 steps), I found that the accuracy was significantly lower when calculating the uncertainty within the 100k-1000k target range using the resulting potential function.

In this process, all parameters remained unchanged except for `DP_INFER_BATCH_SIZE`, yet the potential function showed a drastically different effect.

My questions are: 1. Is this phenomenon normal? 2. If this is possible, what is the principle behind how the `DP_INFER_BATCH_SIZE` parameter changes the potential function?

### DeePMD-kit Version

DeePMD-kit v3.1.1

### Backend and its version

TensorFlow  2.18.0

### Details

DP_INFER_BATCH_SIZE=1024 dp train，  DP_INFER_BATCH_SIZE=102400 dp train
